docker/dev/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM node:lts
+
+# Port for web server
+ENV PORT=3000
+ENV WEB_PORT=8080
```

Comment:
I don't remember there being a `WEB_PORT` environment variable. Do we use that? or does webpack?

Reply:
it's not used it's more for a remember, but il will remove it because the port used with the dev image is 8888 for webpack and 3001 or 3001 with vue

Reply:
You mean 8080, not 8888 right?